<?xml version="1.0"?>
<launch>
	<!-- Global flags -->
	<arg name="obs_srv_cfg" default="$(find uas_offboard_planner)/config/default_dvp.yaml"/>

	<!-- Launch observation server standalone -->
	<rosparam command="load" file="$(arg obs_srv_cfg)"/>
	<node pkg="uas_offboard_planner" type="obs_srv_node" name="obs_srv_node" output="screen" respawn="false" respawn_delay="0" />
	
</launch>
